src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/WindowsFormsProjectSchema.xaml

```diff
@@ -0,0 +1,32 @@
+﻿<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
+<Rule Name="WindowsFormsConfigurationGeneral"
+      DisplayName="WindowsFormsConfigurationGeneral"
```

Comment:
Can we make the name consistent across this PR? Currently it uses both `WindowsFormsConfigurationGeneral` and `WindowsFormsProjectSchema`.

We use "configuration general" for general properties, and this is WinForms-specific. 

I think a more consistent name would be `WindowsFormsConfiguration`.

Reply:
I've renamed a bunch of them. Please check.